app/models/build/utils.rb

```diff
@@ -52,7 +52,18 @@ def fix_version_string(version)
         semVerReg = /#{Regexp.escape(specifier)}\s*(\d+\.)*[x\*]/
 
         version.gsub!(semVerReg) do |match|
-          match.strip[0..-3]
+          new = match.strip[0..-3]
```

Comment:
May we use an identifier other than `new`? To my eyes, this creates confusion about whether we're talking about a constructor or an object when I'm scanning quickly.